src/main/log/.gitignore

```diff
@@ -0,0 +1,2 @@
+/mzmine.log.1
```

Comment:
I think this should be added to the .gitignore file in the root folder

Reply:
After changing the working directory it started to produce these files. Somehow the gitignore is not shown in the folder tree.